.github/workflows/verify-k8s.yml

```diff
@@ -136,8 +136,8 @@ jobs:
           # Reduce the resource requests of Fulcio
           sed -i -e 's,memory: "1G",memory: "100m",g' ${{ github.workspace }}/config/deployment.yaml
           sed -i -e 's,cpu: ".5",cpu: "50m",g' ${{ github.workspace }}/config/deployment.yaml
-          # Switch to the ephemeralca for testing.
-          sed -i -e 's,--ca=googleca,--ca=ephemeralca,g' ${{ github.workspace }}/config/deployment.yaml
+          # Switch to fileca for testing.
+          sed -i -e 's,--ca=googleca,--ca=fileca,g' ${{ github.workspace }}/config/deployment.yaml
```

Comment:
Once the secret is set up and mounted, the `fileca` needs a couple more flags to just figure out where the certs are mounted and the password to decrypt the key. They're as follows

- `--fileca-cert` for the cert path
- `--fileca-key` for the key path
- `--fileca-passwd` for the key password
- `--fileca-watch=true/false` if you want to watch the cert and key path for updates and reload on change

I'm less familiar with how the deployment is configured, but these flags will need to be set somewhere to get fileca working